src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ *  in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see
+ *  an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styledObject = {};
+  const sourceStringArray = sourceString
+    .split(';')
+    .map((item) => item.split(':'));
+
+  for (const item of sourceStringArray) {
+    const [key, value] = item;
+
+    if (value) {
+      styledObject[key.trim()] = value.trim();
+    }
```

Comment:
It would be better to use here the reduce( ) to create the object.

Reply:
> It would be better to use here the reduce( ) to create the object.

Good point, don't use `for of`, this task is for array iteration method practice